CMakeLists.txt

```
@@ -64,13 +30,15 @@ find_package(Z3 REQUIRED)
 add_subdirectory(src)
 
 # add test code
+option(BUILD_QCEC_TESTS "Also build tests for QMAP project")
```

this says QCEC and QMAP for some reason.
```suggestion
option(BUILD_QUSAT_TESTS "Also build tests for QUSAT project")
```